app/models/metasploit/cache/module/target/architecture.rb

```diff
@@ -7,19 +7,22 @@ class Metasploit::Cache::Module::Target::Architecture < ActiveRecord::Base
   # Associations
   #
 
-  # @!attribute architecture
-  #   The architecture supported by the {#module_target}.
-  #
-  #   @return [Metasploit::Cache::Architecture]
+  # The architecture supported by the {#module_target}.
   belongs_to :architecture, class_name: 'Metasploit::Cache::Architecture', inverse_of: :target_architectures
 
-  # @!attribute module_target
-  #   The module target that supports {#architecture}.
-  #
-  #   @return [Metasploit::Cache::Module::Target]
+  # The module target that supports {#architecture}.
   belongs_to :module_target, class_name: 'Metasploit::Cache::Module::Target', inverse_of: :target_architectures
 
   #
+  # Attributes
+  #
+
+  # @!method architecture_id
```

Comment:
Is there a reason these aren't `@!attribute`s? Using `@!method` will not create a setter.